common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -114,4 +115,30 @@ public Matcher lootChestMatcher(Screen screen) {
         return LOOT_CHEST_PATTERN.matcher(
                 WynnUtils.normalizeBadString(ComponentUtils.getUnformatted(screen.getTitle())));
     }
+
+    public Pair<Integer, Integer> getScrollSlots(AbstractContainerScreen<?> gui, boolean scrollUp) {
```

Comment:
Since you get the scrollUp input, you don't need to return a pair but can just return the requested scroll slot itself ...